packages/vuetify/src/labs/VCalendar/VCalendarIntervalEvent.tsx

```diff
@@ -55,8 +58,8 @@ export const VCalendarIntervalEvent = genericComponent()({
         <VSheet
```

Comment:
@blalan @yuwu9145 - I moved out some of the utility classes, but rounded needs to be ultimately removed or specifically added since a user might not have utility classes enabled.